src/Footer.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { createElement } from "react";
+
+const information = [
+  "123 Fake Street, London, E1 4UD",
+  "hello@fakehotel.com",
+  "0123 456789"
+];
+
+function Footer() {
+  return (
+    <ul>
+      {information.map((item, index) => {
+        return <li key={index}> {item} </li>;
+      })}
+      ;
+    </ul>
+  );
```

Comment:
Great job on creating the `Footer` component. 🥳

You might want to make use of the existing css rule in App.css to centre the footer:
```suggestion
  return (
      <div className="footer">
        <ul>
        {information.map((item, index) => {
            return <li key={index}> {item} </li>;
        })}
        ;
        </ul>
    </div>
  );
```